src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -49,6 +51,10 @@ export class DocService {
     return sections[section];
   }
 
+  getApiDocs() {
```

Comment:
IMHO we don't need to add this method here as it does not contain any business logic here: it's just a plain http request therefore it can be done via injecting angular HttpClient in the ApiDocsComponent.

Reply:
+1, it should be handled in dashboard-help.component.html file itself as I suggested here too https://github.com/ceph/ceph/pull/41523/files#r638594005 .